src/dnx.host/RuntimeBootstrapper.cs

```diff
@@ -64,6 +64,8 @@ public static Task<int> ExecuteAsync(string[] args, FrameworkName targetFramewor
 
             // These options were handled in the native code, but got passed through here.
             // We just need to capture them and clean them up.
+            var optionProject = app.Option("--project|-p", "Path to the project.json file or the application folder. Defaults to the current folder if not provided.",
+                CommandOptionType.NoValue);
```

Comment:
I merely wanted this option to be displayed in help. This is because -p should not get out of dnx  since it is converted to --appbase. However there are degenerate cases where --appbase and -p is provided or -p is provided twice where -p can actually get here. I will look a bit more into this code because NoValue feels wrong indeed. Thanks for pointing this out.